MIT.md

```diff
@@ -1,4 +1,4 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2021, [HABEEB EFIAMOTU MUSA OWOLEWA]
```

Comment:
* Kindly create 5 random greetings as mentioned in the project [requirements ](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=create%20a%20table%20for%20storing%20your%20messages%20and%20create%205%20different%20greetings.)

Reply:
I have created the greetings and they are all displaying. @Davidosky007